apps/emqx_management/test/emqx_mgmt_api_ds_SUITE.erl

```diff
@@ -23,8 +23,22 @@
 
 -import(emqx_mgmt_api_test_util, [api_path/1, request_api/2, request_api_with_body/3]).
 
+-define(ON(NODE, BODY), erpc:call(NODE, fun() -> BODY end)).
```

Comment:
This tests will clash with my other PR that aims to reduce the number of scenarios where cluster is needed. I suggest to skip them altogether in this PR.

Reply:
Without a cluster, it'll be hard to test a down site/shard...